build.gradle

```diff
@@ -125,6 +125,7 @@ ext {
 
     options.compilerArgs << "-encoding" << "UTF-8"
     options.compilerArgs += ["--release", String.valueOf(releaseVersion)]
```

Comment:
Could you please try to use `options.compilerArgs += ["--release", String.valueOf(releaseVersion)]` instead of `options.compilerArgs += ["--release", String.valueOf(releaseVersion)]`. The former can set the `sourceCompatibility` automatically (see https://github.com/gradle/gradle/blob/784ffbe7b04e3dd31f4e05e12906efc7a410757f/platforms/jvm/plugins-java-base/src/main/java/org/gradle/api/plugins/JavaBasePlugin.java#L331)

Reply:
typo: `options.release = releaseVersion` -> we should use this instead